src/puppetlabs/jdbc_util/migration.clj

```diff
@@ -22,20 +25,36 @@
   "Migrate 'db' using migratus with a given 'migration-dir'."
   [db migration-dir]
   (let [have-pglogical (pglogical/has-pglogical-extension? db)
-        pg-schema "public"]
+        pg-schema "public"
+        config {:store :database
+                :migration-dir migration-dir
+                :db db
+                :modify-sql-fn (if have-pglogical
+                                 #(pglogical/wrap-ddl-for-pglogical % pg-schema)
+                                 identity)}
+        store (mproto/make-store config)]
     (try
-      (migratus/migrate {:store :database
-                         :migration-dir migration-dir
-                         :db db
-                         :modify-sql-fn (if have-pglogical
-                                          #(pglogical/wrap-ddl-for-pglogical % pg-schema)
-                                          identity)})
+      (log/info (i18n/trs "Starting migrations"))
+      (mproto/connect store)
+      (let [uncompleted-migrations (sort-by mproto/id (migratus/uncompleted-migrations config store))]
+        (doseq [migration uncompleted-migrations]
+          (let [migration-name (migratus/migration-name migration)
+                _ (log/info (i18n/trs "Up {0}" migration-name))
+                result (mproto/migrate-up store migration)]
+            (when-not (or (= :success result) (= :ignore result))
```

Comment:
This will continue migrating on `:ignore`, whereas migratus will stop.

Reply:
Good point.  I should have left that off.